src/ADAL.PCL.Android/Properties/AssemblyInfo.cs

```diff
@@ -39,6 +39,3 @@
 [assembly: Guid("ff47962a-d498-4c63-b7e9-4db3653ad7df")]
 
 // Assembly version information is in file ADAL.Common\CommonAssemblyInfo.cs
-
-[assembly: AssemblyDelaySign(true)]
-[assembly: AssemblyKeyFile(@"..\..\build\272MSSharedLibSN2048.snk")]
```

Comment:
Difficult to know. I don't know of a reason why it would need to be different, so I decided to make the change and check that everything still worked as expected when testing.